draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +920,37 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> First generations of quantum networks will not be store-and-forward
+           networks.
+           <vspace blankLines="1" />
+
+           As described in <xref target="elg"></xref>,
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed links of classical networks.
+           This phenomenological distinction should result in architectural
+           differences between quantum networks and classical networks;
+           quantum networks put link Bell pairs together to an end-to-end
+           Bell pair where classical networks deliver messages from an end
+           to the other end hop by hop.
+           <vspace blankLines="1" />
+
+           Classical networks store data coming from an interface, then forward
+           the data to another appropreate interface.
+           Quantum networks store a couple of half Bell pairs, then
+           execute entanglement swapping instead of forwarding in a data plane.
+           Such quantum networks are "store and swap" networks.
+           In "store and swap" networks, we do not need to care about
+           the order of generating link Bell pairs since Bell pairs are
+           undirected resources. This distinction makes control algorithms and
```

Comment:
```suggestion
           the order in which the Bell pairs were generated since they are
           undirected. This distinction makes control algorithms and
```

Reply:
```suggestion
           the order in which the Bell pairs are generated since they are
           undirected. This distinction makes control algorithms and
```